modules/ROOT/pages/setup.adoc

```diff
@@ -39,7 +44,8 @@ NOTE: To obtain an Anypoint Partner Manager entitlement, please contact your Mul
 
 == Environment Configuration
 
-You'll need to set up and configure your environment before you begin using Partner Manager. Please refer to Anypoint platform documentation to learn more about https://docs.mulesoft.com/access-management/environments[configuring your Anypoint platform environments].
+You'll need to set up and configure your environment before you begin using Partner Manager.
```

Comment:
Sounds like it should be just a list item in prerequisites, with a link in See Also.